Monika After Story/game/dev/dev_exp_previewer.rpy

```diff
@@ -117,8 +123,8 @@ init 999 python:
             5: ("def", "def")
         }
 
-        # image name map
-        IMG_NMAP = {
+        # Map between sprite code letters and image names
+        IMG_NAMES_MAP = {
```

Comment:
this should use the sprite map loaded from the decoder.

Reply:
Done!